streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -195,9 +193,7 @@ public boolean conditionMet() {
                     return accumData.size() >= expectedNumRecords;
                 }
             };
-            final String conditionDetails =
-                "Expecting " + expectedNumRecords + " records from topic " + topic +
-                    " while only received " + accumData.size() + ": " + accumData;
+            final String conditionDetails = "Did not receive all " + expectedNumRecords + " records from topic " + topic;
```

Comment:
For this message and the one above, do we want to leave in the actual number of records received? Might useful for debugging.

Reply:
Unfortunately we cannot, as the passed in string is static and cannot be lazy evaluated in J7.